kbase-extension/static/kbase/js/widgets/appWidgets2/input/customInput.js

```diff
@@ -1,39 +1,23 @@
 define([
     'bluebird',
-    'require',
```

Comment:
Adjusted some things in here while converting its own `pRequire` -> `lang.pRequire`. Mostly either linting or deleting unused code/dependencies.